libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -487,6 +511,13 @@ DNNReturnType ff_dnn_execute_model_ov(const DNNModel *model, const char *input_n
         return DNN_ERROR;
     }
 
+    if (!ov_model->exe_network || !ov_model->infer_request) {
+        if (init_model_ov(ov_model) != DNN_SUCCESS) {
+            av_log(ctx, AV_LOG_ERROR, "Failed init OpenVINO exectuable network or inference request\n");
+            return DNN_ERROR;
+        };
+    }
+
```

Comment:
two copies of the same code, how about to move into function execute_model_ov

Reply:
OK, will check it here tomorrow

Reply:
just realized that it could not be moved to execute_model_ov, we need to call it at the beginning of get_output_ov, ff_dnn_execute_model_ov and ff_dnn_execute_model_async_ov